### DISCUSSION TITLE
Focus Control Problems With Q6215-LE using ONVIF Protocol

### DISCUSSION THREAD

**Discussion by haydenm315** [AxisCommunications/discussions#240] [ONVIF]
I'm running into hurdles implementing manual focus lens control, using the ONVIF protocol.
[Focus_Stop.txt](https://github.com/AxisCommunications/discussions/files/11435597/Focus_Stop.txt)

1. The camera lens does not respect the http://www.onvif.org/ver20/imaging/wsdl/FocusStop command.

2.  If you set the speed to a value besides 1 or -1, manual focus near commands don't work. Setting the speed also doesn't affect speed of continuous focus commands.

3. You can issue manual focus commands while in auto focus mode. Not a big deal, but inconsistent with some cameras that won't respond to manual focus commands, except when in manual focus mode.

The camera not observing the focus stop command is the largest issue. I've attached the xml communication for this command. The camera responds with a success 200 code and a StopResponse, which indicates success, however the camera's lens continues to focus.

Verified this behavior with the latest firmware. 11.4.63

**Comment by bsriramprasad** [comment]
Issue seems to be corrected in later firmwares, can you confirm if you still see the issue in 11.11.116 - AXIS OS LTS 2024?
